modules/blocks.js

```diff
@@ -1269,11 +1770,32 @@ Blocks.prototype.deleteAfterBlock = function (blockId, cb) {
 	});
 };
 
+/**
+ * Sandbox API wrapper
+ *
+ * @public
+ * @async
+ * @method sandboxApi
+ * @param  {string}   call Name of the function to be called 
+ * @param  {Object}   args Arguments
+ * @return {Function} cb Callback function
+ */
 Blocks.prototype.sandboxApi = function (call, args, cb) {
 	sandboxHelper.callMethod(Blocks.prototype.shared, call, args, cb);
 };
 
-// Events
+/**
+ * EVENTS
+ */
+
+/**
+ * Handle newly received block
+ *
+ * @public
+ * @method onReceiveBlock
+ * @listens module:transport~event:receiveBlock
+ * @param  {block}   block New block
+ */
```

Comment:
So it should be `@return {undefined}`? When I ommit `@return` tag compiler should assume that the function returns `undefined`.